collections/_blogs/2020-06-25-SMI-conformance-testing-with-meshery.md

```diff
@@ -11,52 +11,52 @@ permalink: /blog/smi-conformance-testing-with-meshery
+All these questions will be answered by SMI conformance tests. The biggest benefit of your service mesh conforming to SMI is that it makes building tools an easier process.Also, one time development of any tool would in turn support all the meshes that conform to SMI.
 
-And the biggest beneifit of mesh conforming to SMI would be that:- Building tools for SMI compatible mesh would be easier, and one time development of the tool would in turn support all the meshes that conforms to SMI.
+Sounds fun, right? Let's dig deeper into the SMI conformance project and find out.
 
-Sounds fun, right? Well let's dig deeper into the SMI conformance project.
 
 ### Meshery'ing with Conformance Tests
 
-**[Meshery](/meshery)** is the **service mesh management plane**, it supports all the popular meshes, how to manage them, apply custom or recommended configuration, tests for checking whether meshes are compatible or not, performing performance tests for mesh and lots more. The SMI conformance testing required performance testing capabiliy, load generation (meshery is about to support distributed load generation as well), and other functionality such that these conformance tests can be easily used in pipeline of all the popular service mesh. Mesheryctl does have a perf command that can be used in pipeline of service meshes, I aim at making such capability for SMI conformance too. 
+**[Meshery](/meshery)** is a **service mesh management plane**. It supports all the popular meshes, teaches you how to manage them, assists you in applying custom or recommended configurations, tests for  compatibility, performs performance tests for meshes and a lot more. The SMI conformance testing requires performance testing capabilities, load generation (Meshery is about to support distributed load generation as well), and other functionalities such that these conformance tests can be easily used in the pipelines of all the popular service meshes. mesheryctl does have a perf command that can be used in the pipelines of service meshes. I aim at making such capabilities for SMI conformance as well.
 
-Ok, I hope by this time you know what SMI is, what SMI conformance mean, how meshery's amazing engineering can be utilized for conformance tests, but there might be questions like how these tests were written, how do we validate the traffic, what tools do we use, let's see everything in the bigger picture.
+As you have made your way halfway through the post (thank you for your patience), you should now be aware of SMI, its conformance practices and how Meshery’s incredible engineering can be utilized for conformance tests. We can now tackle the larger questions and hope to see the bigger picture.
```

Comment:
some details lost in this rephrasing that engineers (who are the target audience here) will find interesting - "questions like how these tests were written, how do we validate the traffic, what tools do we use, let's see everything in the bigger "